### PR TITLE
Feature/sc 190307 add template manual input on attribute search connector

### PR DESCRIPTION
**Pull request by alexbourret**
Story details: https://app.shortcut.com/dataiku/story/190307

### CODE REVIEW

**Review by liamlynch-data — COMMENTED** [review]
Looks mainly fine to me and the features in the card all work fine. 
I just have a couple of questions/suggestions about some of the other stuff in the PR

File: python-lib/osisoft_plugin_common.py [hunk 1]
```diff
@@ -372,29 +382,43 @@ def get_max_count(config):
...
     if isinstance(timestamp, int) or isinstance(timestamp, float):
         return True
     return timestamp.replace(".", "", 1).isdigit()
 
 
+def is_iso8601(timestamp):
+    # https://stackoverflow.com/questions/41129921/validate-an-iso-8601-datetime-string-in-python
+    if not isinstance(timestamp, str):
+        return False
+    return match_iso8601(timestamp) is not None
```
[the remark this answers is not in the document]
Comment (alexbourret) [reply python-lib/osisoft_plugin_common.py]: 👍 added in [6bd583c](https://github.com/dataiku/dss-plugin-osisoft/pull/42/commits/6bd583cae2eb2e761a7a0bea2fd2c67d433e6bac)

File: python-lib/osisoft_plugin_common.py [hunk 2]
```diff
@@ -289,6 +293,8 @@ def is_filtered_out(item, filters=None):
 def is_server_throttling(response):
     if response is None:
         return True
+    if response.status_code == 409 and not response_has_retry_header(response):
+        return True
```
[the remark this answers is not in the document]
Comment (alexbourret) [reply python-lib/osisoft_plugin_common.py]: can't remember why... so I removed that case in [22be5da](https://github.com/dataiku/dss-plugin-osisoft/pull/42/commits/22be5daf28fa7a9cdc019393c45992d2f3aea257)

**Review by liamlynch-data — APPROVED** [review]
LGTM now 👍

**Pull request merged**